### ISSUE TITLE
Cleanup of descriptions and keywords

### ISSUE THREAD

**Issue by evanshultz** [KiCad/kicad-symbols#1617]
While reviewing https://github.com/KiCad/kicad-symbols/pull/1616, I looked at the DCM file changes. Here is the part submitted and the two after it:
```
$CMP Si7141DP
D 60A Id, 20V Vds, 1.9 mOhm Ron, PowerPAK-8
K P-Channel MOSFET
F https://www.vishay.com/docs/65596/si7141dp.pdf
$ENDCMP
#
$CMP Si7336ADP
D 30A Id, 30V Vds, 2.4 mOhm Ron, PowerPAK-8
K N-Channel MOSFET
F www.vishay.com/docs/73152/si7336adp.pdf
$ENDCMP
#
$CMP SiA449DJ
D -12A Id, -30V Vds, P-Channel MOSFET, PowerPAK SC70-6
K P-Channel MOSFET
F http://www.vishay.com/docs/62644/sia449dj.pdf
$ENDCMP
#
```

You can see one difference in that the type of transistor (NPN, PNP, N-channel, P-channel, IGBT, etc.) is in the description on one but not the others. And the keywords contain just the transistor type.

Now that we will be heading towards v6, perhaps it's a good time to think through our descriptions (I'm using transistors above but the scope can be library-wide) and figure out if there are more rules and structure we should put in place for descriptions and keywords?

Lastly, I want to be clear that the contributor did everything right and this PR is fine. It's a framework and consistency issue I'm surfacing here.

**Comment by myfreescalewebpage** [comment]
I agree with you @evanshultz, if some description/keywords guidelines can be added to very large families, such as transistors, regulators, etc to help search when making a schematic, that's a good idea !!!
